config/env/index.js

```diff
@@ -8,6 +8,7 @@
   UPTIMA_NODE_ENV: NODE_ENV
 } = process.env;
 
+
```

Comment:
Codacy found an issue: [More than 1 blank line not allowed.](https://app.codacy.com/gh/UptimaNGR/uptima-backend/pullRequest?prid=6048355)